first_order_model/run.py

```diff
@@ -43,8 +43,12 @@
         config = yaml.load(f)
 
     if opt.mode == "reconstruction":
-        log_dir = os.path.dirname(opt.checkpoint)
-        log_dir = os.path.join(log_dir, 'reconstruction' + '_' + opt.experiment_name)
+        loc = config['train_params'].get('shrunk_gen', None)
```

Comment:
what is this `loc` doing? Can you use a better variable name? Is there a need for a diifferent path name than what I have set up in this case? This is going to overwrite/fill the checkpoint directory likely, so would rather have a separate folder for reconstruction.